tests/e2e/test_kg_construction_e2e.py

```diff
@@ -0,0 +1,93 @@
+#  Copyright (c) "Neo4j"
+#  Neo4j Sweden AB [https://neo4j.com]
+#  #
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#  #
+#      https://www.apache.org/licenses/LICENSE-2.0
+#  #
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import neo4j
+import pytest
+from neo4j_genai.kg_construction.kg_writer import Neo4jWriter
+from neo4j_genai.kg_construction.types import (
+    Neo4jEmbeddingProperty,
+    Neo4jGraph,
+    Neo4jNode,
+    Neo4jProperty,
+    Neo4jRelationship,
+)
+
+
+@pytest.mark.asyncio
+@pytest.mark.usefixtures("setup_neo4j_for_kg_construction")
+async def test_kg_writer(driver: neo4j.Driver) -> None:
+    start_node = Neo4jNode(
+        id="1",
+        label="Document",
+        properties=[Neo4jProperty(key="chunk", value=1)],
+        embedding_properties=[
+            Neo4jEmbeddingProperty(key="vectorProperty", value=[1.0, 2.0, 3.0])
+        ],
+    )
+    end_node = Neo4jNode(
+        id="2",
+        label="Document",
+        properties=[Neo4jProperty(key="chunk", value=2)],
+        embedding_properties=[
+            Neo4jEmbeddingProperty(
+                key="vectorProperty",
+                value=[1.0, 2.0, 3.0],
+            )
+        ],
+    )
+    relationship = Neo4jRelationship(
+        start_node_id="1", end_node_id="2", label="NEXT_CHUNK"
+    )
+    graph = Neo4jGraph(nodes=[start_node, end_node], relationships=[relationship])
+
+    neo4j_writer = Neo4jWriter(driver=driver)
+    await neo4j_writer.run(graph=graph)
+
+    query = """
+    MATCH (a:Document {id: 1})-[r:NEXT_CHUNK]-(b:Document {id: 2})
```

Comment:
Is it intentional that the relationship `-[r:NEXT_CHUNK]-` is bidirectional?

Reply:
No that's a mistake, I'll fix it